lib/generate-config.js

```diff
@@ -46,9 +46,26 @@ module.exports = function(primaryOptions, secondaryOptions) {
     presetOptions = primaryOptions.presetOptions;
   }
 
+  var implicitComponents = secondaryOptions && secondaryOptions.implicitComponents !== undefined ?
+    secondaryOptions.implicitComponents : primaryOptions && primaryOptions.implicitComponents;
+  implicitComponents = implicitComponents === undefined ? false : implicitComponents;
```

Comment:
This is too many operators without blocks ... can you break this up and make it more legible please?